projects/demo/src/app/grid/grid.demo.scss

```
@@ -8,6 +8,8 @@
 @import '../_utils/clarity-variables';
 
 .clr-example {
+  $border-color: hsl(218, 14%, 30%);
```

Can we use token here?

```suggestion
  $border-color: tokens.$cds-alias-object-interaction-border-color;
```

---

Yes, totally. Added it.

---

Added in [7079953](https://github.com/vmware-clarity/ng-clarity/pull/1457/commits/7079953691033cf5a38266ce82c7f51be0f19485)